test/js/bun/glob/scan.test.ts

```diff
@@ -445,6 +446,15 @@ test("glob.scan('.')", async () => {
   expect(entries).toContain("README.md");
 });
 
+test("absolute path pattern should ignore cwd and start at the proper path", async () => {
```

Comment:
can we get more tests for this? is there a way to run all the tests that run against relative paths to also run against absolute paths?